employee-management/webapp/controller/Detail.controller.js

```diff
@@ -0,0 +1,30 @@
+sap.ui.define([
+    "sap/ui/core/mvc/Controller"
+], function (Controller) {
+    "use strict";
+
+    return Controller.extend("ca.deloitte.hr.employeemanagement.controller.Detail", {
+        onInit: function () {
+            var oOwnerComponent = this.getOwnerComponent();
+
+            this.oRouter = oOwnerComponent.getRouter();
+            this.oModel = oOwnerComponent.getModel();
+
+            this.oRouter.getRoute("EmployeeList").attachPatternMatched(this._onEmployeeMatched, this);
```

Comment:
Please try to keep a standard for the route name. Example:
EmployeeList, EmployeeDetail, Etc. or list, detail.
Both are correct, however, I would prefer a short version: list, detail, create, etc.